app/api/v2/blueprints/version.py

```diff
@@ -0,0 +1,13 @@
+from dataclasses import asdict
+
+from flask import Blueprint, jsonify
+
+from app.api.common.api_version import ApiVersion
+
+VERSION = ApiVersion(full="2", release_date="2024-01", deprecated=False)
```

Comment:
shouldn't the version be coming from package.json instead of being hardcoded into the code?

Reply:
Yes it should. Ultimately, after discussing with you and @ThomasCardin, having multiple versions up in the same instance might not be what we want. I'll drop this branch and come up with a solution more in line with our vision of versioning.

Reply:
@k-allagbe should collaborate on an ADR to flesh out the requirements and design for managing semantic versions of our components but also how we deploy them in our development environments with @ThomasCardin ; probably within https://github.com/ai-cfia/dev-rel-docs/pull/80/files